src/components/inputComponents/InputItemName.js

```diff
@@ -0,0 +1,5 @@
+export default function InputItemName() {
+  return (
+    <input type="text" id="name" name="itemName" placeholder="cake name" />
```

Comment:
I'd name this input `cakeName` or something like that, not `itemName`

Reply:
Naming is tricky :)